app/Controllers/feedController.php

```diff
@@ -647,6 +647,126 @@ public function deleteAction() {
 		}
 	}
 
+	/**
+	 * This action force to clear cache of a feed.
+	 *
+	 * Parameters are:
+	 *   - id (mandatory - no default): Feed ID
+	 *
+	 */
+	public function clearcacheAction() {
+		//Get Feed.
+		$id = Minz_Request::param('id');
+		
```

Comment:
Please trim whitespace at the end of all lines.
Cf. https://travis-ci.org/FreshRSS/FreshRSS/jobs/639279131

Reply:
Done.